repl/convert_ast.py

```diff
@@ -7,19 +7,64 @@
 import qwerty_ast
 #################### COMMON CODE FOR BOTH @QPU AND @CLASSICAL DSLs ####################
 
+
 def convert_to_qwerty(py_ast):
     print(ast.dump(py_ast, indent=4))
-    value = unpack_ast(py_ast)
-    if value is None:
-        return "Invalid Expression: Not a qubit literal or Tensor"
-    tensor = qwerty_ast.NodeBox.new_qubit_tensor(py_ast.body[0].value.value)
-    return tensor.get_tensor()
-
-def unpack_ast(ast):
+    if not isinstance(py_ast, (ast.Interactive, ast.Module, ast.Expression)):
+        return "Invalid AST"
+    
+    value = py_ast.body[0]
```

Comment:
Please forgive my paranoia but here are some things we should consider:

1. It looks like `Expression` has a `body` that is not a list: https://docs.python.org/3/library/ast.html. I would either remove `ast.Expression` from the `isinstance()` call above or account for this somehow
2. That aside, I think we should check here that `len(py_ast.body) == 1` before we do `py_ast.body[0]`. I believe parsing something like `foo; bar` will result in a body with two nodes in it